ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/constants/ServerConfig.java

```diff
@@ -7,12 +7,16 @@
 @ConfigurationProperties(prefix = "ldes-server")
 public class ServerConfig {
 	private static final String DEFAULT_COMPACTION_DURATION = "P7D";
-	private static final String DEFAULT_RETENTION_CRON = "*/10 * * * * *";
+	private static final String DEFAULT_BACKGROUND_CRON = "*/10 * * * * *";
```

Comment:
I am not sure if we want this default to be so short. I'll create a poll on slack.

Reply:
can you make this once per day as discussed on slack?